src/test/sandbox/default.jsonc

```diff
@@ -0,0 +1,10 @@
+{
+    // Inline comment
+    "foo": "default",
+    /*
+        Block comment
+    */
+    "bar": 1,
+    "baz": 2,
+    "defaultConfig": true
+}
```

Comment:
Missing newline

Reply:
Fixed. Was a 1:1 copy of an already present `default.json`

Reply:
Whoopsie, my bad.

Reply:
Thanks.